lib/services/suppressed-violations-manager.js

```diff
@@ -0,0 +1,275 @@
+"use strict";
```

Comment:
Please include our standard file header here. You can look at eslint/eslint.js as an example.